### ISSUE TITLE
licenseLocation is enforced even when using OpenClover

### ISSUE THREAD

**Issue by Alex-Vol** [bmuschko/gradle-clover-plugin#85]
The plugin enforces a default location "$projectDir/clover.license" to be an existing file even when using OpenClover. With the introduction of OpenClover the licenseLocation should not assume a default location but  instead have no assigned value. If a value is assigned we should validate it is an existing file otherwise do nothing.
```
:cloverAggregateDatabases FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':cloverAggregateDatabases'.
> File '/home/user/myProject/clover.license' specified for property 'licenseFile' does not exist.
```

**Comment by Alex-Vol** [comment]
This workaround exists to alleviate the issue since OpenClover does not expect or need an actual license file
```
clover {
    licenseLocation = File.createTempFile('clover', '.license').absolutePath
}
```

**Comment by Alex-Vol** [comment]
Fixed, will be part of the 2.1.1 milestone
